pandas/core/frame.py

```diff
@@ -2180,6 +2180,7 @@ def to_html(
         header=True,
         index=True,
         na_rep="NaN",
+        encoding=None,
```

Comment:
shouldn't this actually default to 'utf-8'?

Reply:
I actually asked for this to be done since the base class handles it in the `get_buffer` method. It also matches the interface exposed by the CSVFormatter which has similar logic in its constructor. 

Not tied it if you disagree just figured consistency and less duplication of logic would be preferable

Reply:
as it is now, it's also consistent with to_latex. so I _think_ OK to leave this as is.